iroh-p2p/src/config.rs

```diff
@@ -158,7 +159,10 @@ impl Default for Libp2pConfig {
             .collect();
 
         Self {
-            listening_multiaddr: "/ip4/0.0.0.0/tcp/4444".parse().unwrap(),
+            listening_multiaddrs: vec![
+                "/ip4/0.0.0.0/tcp/4444".parse().unwrap(),
+                "/ip4/0.0.0.0/udp/4445/quic".parse().unwrap(),
```

Comment:
@elenaf9 this fails with `Error: Multiaddr is not supported: /ip4/0.0.0.0/udp/4445/quic` :/

Reply:
Yes that's because the `quic` codepoint is now interpreted as QUIC version draft-29. See libp2p/rust-libp2p#3133. 
Use `/ip4/0.0.0.0/udp/4445/quic-v1` instead.